src/assets/i18n/messages.de.json

```diff
@@ -0,0 +1,198 @@
+  "cbe16a57be414e84b6a68309d08fad894df797d6": "Verschlüsselung verwenden",
+  "0c1875a79b7ecc792cc1bebca3e063e40b5764f9": "Dateipfad zum Zertifikat",
+  "736551b93461d2de64b118cf4043eee1d1c2cb2c": "Dateipfad zum Zertifikatsschlüssel",
+  "4e3120311801c4acd18de7146add2ee4a4417773": "Abonnements erlauben",
+  "4bee2a4bef2d26d37c9b353c278e24e5cd309ce3": "Abonnements Basispfad",
+  "bc9892814ee2d119ae94378c905ea440a249b84a": "Basispfad für Videos von abonnierten Kanälen und Wiedergabelisten. Dieser ist relativ zum Stammordner von YTDL-Material.",
+  "5bef4b25ba680da7fff06b86a91b1fc7e6a926e3": "Prüfintervall",
+  "0f56a7449b77630c114615395bbda4cab398efd8": "Einheit ist Sekunden, nur Zahlen sind erlaubt.",
+  "78e49b7339b4fa7184dd21bcaae107ce9b7076f6": "Youtube-DL Archiv verwenden",
+  "fa9fe4255231dd1cc6b29d3d254a25cb7c764f0f": "Mit der Archivfunktion",
+  "09006404cccc24b7a8f8d1ce0b39f2761ab841d8": "werden Informationen über Videos, welche durch ein Abonnement heruntergeladen wurden, in einem Textdokument festgehalten. Diese befinden sich in dem Archiv Unterverzeichnis vom Abonnementsordner.",
+  "29ed79a98fc01e7f9537777598e31dbde3aa7981": "Dadurch können Videos permanent gelöscht werden, ohne das Abonnement beenden zu müssen. Außerdem kann dadurch aufgezeichnet werden, welche Videos heruntergeladen wurden. Z. B. im Falle eines Datenverlusts.",
+  "27a56aad79d8b61269ed303f11664cc78bcc2522": "Design",
+  "ff7cee38a2259526c519f878e71b964f41db4348": "Standard",
+  "adb4562d2dbd3584370e44496969d58c511ecb63": "Dunkel",
+  "7a6bacee4c31cb5c0ac2d24274fb4610d8858602": "Designänderung erlauben",
+  "fe46ccaae902ce974e2441abe752399288298619": "Sprache",
+  "82421c3e46a0453a70c42900eab51d58d79e6599": "Allgemein",
+  "ab2756805742e84ad0cc0468f4be2d8aa9f855a5": "Audio Basispfad",
+  "c2c89cdf45d46ea64d2ed2f9ac15dfa4d77e26ca": "Dateipfad für Audio-Downloads. Dieser ist relativ zum Stammordner von YTDL-Material.",
+  "46826331da1949bd6fb74624447057099c9d20cd": "Video Basispfad",
+  "17c92e6d47a213fa95b5aa344b3f258147123f93": "Dateipfad für Video-Downloads. Dieser ist relativ zum Stammordner von YTDL-Material.",
+  "6b995e7130b4d667eaab6c5f61b362ace486d26d": "Globale benutzerdefinierte Argumente für Downloads auf der Startseite. Argumente werden durch zwei Kommas voneinander getrennt: ,,",
+  "0ba25ad86a240576c4f20a2fada4722ebba77b1e": "Downloader",
+  "61f8fd90b5f8cb20c70371feb2ee5e1fac5a9095": "Titel der Kopfzeile",
+  "78d3531417c0d4ba4c90f0d4ae741edc261ec8df": "Dateimanager aktivieren",
+  "a5a1be0a5df07de9eec57f5d2a86ed0204b2e75a": "Download-Manager aktivieren",
+  "c33bd5392b39dbed36b8e5a1145163a15d45835f": "Qualitätsauswahl erlauben",
+  "bda5508e24e0d77debb28bcd9194d8fefb1cfb92": "Nur Download Modus",
+  "09d31c803a7252658694e1e3176b97f5655a3fe3": "Multi-Download Modus erlauben",
+  "d8b47221b5af9e9e4cd5cb434d76fc0c91611409": "Einstellungen durch Pin schützen",
+  "f5ec7b2cdf87d41154f4fcbc86e856314409dcb9": "Neuen PIN festlegen",
```

Comment:
```suggestion
  "f5ec7b2cdf87d41154f4fcbc86e856314409dcb9": "Neuen Pin festlegen",
```

Reply:
Pin statt PIN? Es ist ein Akronym. Von daher... Eigentlich [PIN](https://www.duden.de/rechtschreibung/PIN)

Reply:
Dann sollte Pin in Zeile 93 auch PIN sein.

Reply:
Beide sind jetzt PIN.

Reply:
So viele Deutsche hier, doch die Deutschen brauchen eine Übersetzung eigentlich am wenigsten :D

Reply:
Ich hab schon gesagt, dass ich das Projekt auch übersetzen würde, aber es vermutlich kaum anrühren würde nach dem ersten Commit, weil ich Software wie diese in Englisch nutze. XD

YT selbst ist auch EN, sowie 90% des Contents den ich schaue. der Rest ist eher JP als DE. :,D

Aber ist doch eigentlich super umso mehr die Software nutzen können, desto besser. Füllt definitiv ne Lücke auf dem Softwaremarkt. :)

Reply:
Da hast du recht, ich werde die Software auch weiterhin in Englisch nutzen. Die Deutschen sind durch ihre gute englische Ausbildung aber meist nicht auf Übersetzungen angewiesen.